app/controllers/membership_applications_controller.rb

```diff
@@ -68,6 +68,11 @@ def information
 
   end
 
+  def destroy
+    @membership_application.destroy
+    redirect_to membership_applications_url, notice: t('membership_applications.application_deleted')
```

Comment:
it *WAS* correct, when you added this comment. Then Ashley changed it. Now it is not correct. =) 
it is (now)
`application_deleted: Ansökan raderat.`
It was, and should be
`application_deleted: Ansökan raderad.`